src/main/resources/webcert/features.yaml

```diff
@@ -712,3 +718,12 @@ pilots:
         intygstyper:
           - *ki050 # LISJP
           - *ki120 # AG7804
+          - *ki100 # AF00213
```

Comment:
You can remove AF00213 as it is not in production yet.

Reply:
Done!